examples/webgl_materials_cubemap_render_to_mipmaps.html

```diff
@@ -0,0 +1,278 @@
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+	<body>
+
+		<div id="container"></div>
+		<div id="info">
+			<a href="https://threejs.org" target="_blank" rel="noopener">three.js</a> - rendering to cubemap mip levels demo.<br/>
+			Left: original cubemap<br/>
+			Right: generated cubemap<br/>
+		</div>
+
+		<!-- Import maps polyfill -->
+		<!-- Remove this when import maps will be widely supported -->
+		<script async src="https://unpkg.com/es-module-shims@1.6.3/dist/es-module-shims.js"></script>
+
+		<script type="importmap">
+			{
+				"imports": {
+					"three": "../build/three.module.js",
+					"three/addons/": "./jsm/"
+				}
+			}
+		</script>
+
+		<script type="module">
+
+			import * as THREE from 'three';
+			import { OrbitControls } from 'three/addons/controls/OrbitControls.js';
+
+			let container;
+			let camera, scene, renderer;
+
+			const CubemapFilterShader = {
+				uniforms: {
+					cubeTexture: { value: null },
+					mipIndex: { value: 0 },
+				},
+
+				vertexShader: /* glsl */ `
+
+					varying vec3 vWorldDirection;
+
+					#include <common>
+
+					void main() {
+						vWorldDirection = transformDirection(position, modelMatrix);
+						#include <begin_vertex>
+						#include <project_vertex>
+						gl_Position.z = gl_Position.w; // set z to camera.far
+					}
+
+					`,
+
+				fragmentShader: /* glsl */ `
+
+					uniform samplerCube cubeTexture;
+					varying vec3 vWorldDirection;
+
+					uniform float mipIndex;
+
+					#include <common>
+
+					void main() {
+						vec3 cubeCoordinates = normalize(vWorldDirection);
+						
+						// Colorize mip levels
+						vec4 color = vec4(1.0, 0.0, 0.0, 1.0);
+						if (mipIndex == 0.0) color.rgb = vec3(1.0, 1.0, 1.0);
+						else if (mipIndex == 1.0) color.rgb = vec3(0.0, 0.0, 1.0);
+						else if (mipIndex == 2.0) color.rgb = vec3(0.0, 1.0, 1.0);
+						else if (mipIndex == 3.0) color.rgb = vec3(0.0, 1.0, 0.0);
+						else if (mipIndex == 4.0) color.rgb = vec3(1.0, 1.0, 0.0);						
+
+						gl_FragColor = textureCube(cubeTexture, cubeCoordinates, 0.0) * color;
+					}
+
+					`,
+			};
+
+
+			init();
+			animate();
+
+
+			async function loadCubeTexture( urls ) {
+
+				return new Promise( function ( resolve ) {
+
+					new THREE.CubeTextureLoader().load( urls, function ( cubeTexture ) {
+
+						resolve( cubeTexture );
+
+					} );
+
+
+				} );
+
+			}
+
+			function allocateCubemapRenderTarget( cubeMapSize ) {
+
+				const params = {
+					magFilter: THREE.LinearFilter,
+					minFilter: THREE.LinearMipMapLinearFilter,
+					generateMipmaps: false,
+					type: THREE.HalfFloatType,
+					format: THREE.RGBAFormat,
+					colorSpace: THREE.LinearSRGBColorSpace,
+					depthBuffer: false,
+				};
+
+				const rt = new THREE.WebGLCubeRenderTarget( cubeMapSize, params );
+
+				const mipLevels = Math.log( cubeMapSize ) * Math.LOG2E + 1.0;
+				for ( let i = 0; i < mipLevels; i ++ ) rt.texture.mipmaps.push( {} );
```

Comment:
@mrdoob Explicitly defining the mipmaps of the cube render target like demonstrated here triggers the new code path. So I don't think existing app level code is affected by this change. I also this is a nice solution since in this way the developer can freely configure the mipmap chain.